l1-contracts/src/core/libraries/HeaderLib.sol

```diff
@@ -179,15 +160,14 @@ library HeaderLib {
     // Reading totalFees
     header.totalFees = uint256(bytes32(_header[0x0248:0x0268]));
 
-    if (hasTotalManaUsed) {
-      header.totalManaUsed = uint256(bytes32(_header[0x0268:0x0288]));
-    }
+    // Reading totalManaUsed
+    header.totalManaUsed = uint256(bytes32(_header[0x0268:0x0288]));
 
     return header;
   }
 
   function toFields(Header memory _header) internal pure returns (bytes32[] memory) {
-    bytes32[] memory fields = new bytes32[](24);
+    bytes32[] memory fields = new bytes32[](25);
```

Comment:
This can read from the constants no?

Reply:
Yes, but they are also deleted in the blob pr, so did not really bother too much with creating a constant and such here.